src/Features/Core/Portable/Wrapping/InitializerExpression/AbstractInitializerExpressionWrapper`2+InitializerExpressionCodeActionComputer.cs

```diff
@@ -1,210 +0,0 @@
-// The .NET Foundation licenses this file to you under the MIT license.
-// See the LICENSE file in the project root for more information.
-
-using System.Collections.Immutable;
-using System.Threading;
-using System.Threading.Tasks;
-using Microsoft.CodeAnalysis.Editing;
-using Microsoft.CodeAnalysis.Options;
-using Microsoft.CodeAnalysis.PooledObjects;
-using Microsoft.CodeAnalysis.Shared.Extensions;
-using Microsoft.CodeAnalysis.Text;
-using Microsoft.CodeAnalysis.Wrapping.SeparatedSyntaxList;
-using Roslyn.Utilities;
-
-namespace Microsoft.CodeAnalysis.Wrapping.InitializerExpression
-{
-    internal abstract partial class AbstractInitializerExpressionWrapper<TListSyntax, TListItemSyntax>
-    {
-        /// <summary>
-        /// Class responsible for actually computing the entire set of code actions to offer the user.
-        /// </summary>
-        private sealed class InitializerExpressionCodeActionComputer : AbstractSeparatedListCodeComputer<AbstractInitializerExpressionWrapper<TListSyntax, TListItemSyntax>>
-        {
-            private readonly SyntaxTrivia _braceIndentationTrivia;
-            private readonly bool _doMoveOpenBraceToNewLine;
-
-            public InitializerExpressionCodeActionComputer(
-                AbstractInitializerExpressionWrapper<TListSyntax, TListItemSyntax> service,
-                Document document, SourceText sourceText, DocumentOptionSet options,
-                TListSyntax listSyntax, SeparatedSyntaxList<TListItemSyntax> listItems,
-                bool doMoveOpenBraceToNewLine, CancellationToken cancellationToken)
-                : base(service, document, sourceText, options, listSyntax, listItems, cancellationToken)
-            {
-                var generator = SyntaxGenerator.GetGenerator(OriginalDocument);
-
-                _braceIndentationTrivia = generator.Whitespace(GetBraceTokenIndentation());
-                _doMoveOpenBraceToNewLine = doMoveOpenBraceToNewLine;
-            }
-
-            private string GetBraceTokenIndentation()
-            {
-                var previousToken = _listSyntax.GetFirstToken().GetPreviousToken();
-
-                // Block indentation is the only style that correctly indents across all initializer expressions
-                return GetIndentationAfter(previousToken, Formatting.FormattingOptions.IndentStyle.Block);
-            }
-
-            protected sealed override async Task<ImmutableArray<WrappingGroup>> ComputeWrappingGroupsAsync()
-            {
-                using var _ = ArrayBuilder<WrappingGroup>.GetInstance(out var result);
-                await AddWrappingGroupsAsync(result).ConfigureAwait(false);
-                return result.ToImmutableAndClear();
-            }
-
-            protected sealed override ImmutableArray<Edit> GetWrapEachEdits(
-                WrappingStyle wrappingStyle, SyntaxTrivia indentationTrivia)
-            {
-                using var _ = ArrayBuilder<Edit>.GetInstance(out var result);
```

Comment:
all this logic was basically teh same as normal separated wrapping, just with some tweaks about how hte start/end brace are handled.  so this merged back into that, and that code was updated to be hte same as before, just with specialization around start/end brace.